tests/xtimer_usleep/main.c

```diff
@@ -14,35 +14,38 @@
  * @brief       xtimer_usleep test application
  *
  * @author      Francisco Acosta <francisco.acosta@inria.fr>
+ * @author      Martine Lenders <m.lenders@fu-berlin.de>
  * @}
  */
 
+#include <inttypes.h>
 #include <stdio.h>
 
 #include "xtimer.h"
 #include "timex.h"
 
-#define ONE_SEC_SLEEP     (1 * US_PER_SEC)
-#define FIVE_SEC_SLEEP    (5 * US_PER_SEC)
-#define TEN_SEC_SLEEP     (10 * US_PER_SEC)
+#define RUNS                (5U)
+#define SLEEP_TIMES_NUMOF   (sizeof(sleep_times) / sizeof(sleep_times[0]))
+
+static const uint32_t sleep_times[] = { 48028, 73182, 84180, 6978, 52673 };
```

Comment:
Random timing values for fun ? :)
I still prefer round values if someone wants to run 'make term' as it is easier to check if values match.

Reply:
True. Will fix. Though random values are also great to spot strange behavior ;-)

Reply:
Done, used the values proposed by @haukepetersen in https://github.com/RIOT-OS/RIOT/pull/7818#issuecomment-339003723. Also found a little bug in my testing script doing this ;-)